tests/core/test_processor.py

```diff
@@ -664,9 +665,17 @@ async def test_handle_message_with_session_start(
             ],
         ),
         SlotSet(entity, slot_2[entity]),
+        # DefinePrevUserUtteredFeaturization(False),
+        ActionExecuted("utter_greet"),
+        BotUttered(
+            "hey there post-session start hello!",
+            # metadata={"utter_action": "utter_greet"},
```

Comment:
What's the reason for keeping these comments in here? Not sure what you are trying to communicate with these.